src/components/Avatar.tsx

```diff
@@ -13,55 +12,53 @@ import Icon from './Icon';
 import * as Expensicons from './Icon/Expensicons';
 import Image from './Image';
 
-const propTypes = {
+type AvatarProps = {
     /** Source for the avatar. Can be a URL or an icon. */
-    source: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+    source?: AvatarSource;
 
     /** Extra styles to pass to Image */
     // eslint-disable-next-line react/forbid-prop-types
-    imageStyles: PropTypes.arrayOf(PropTypes.object),
+    imageStyles?: ViewStyle[];
```

Comment:
I wanted to comment to change it to `StyleProp<ViewStyle>`, but looking at the implementation I think it's fine to leave as is, to avoid any regressions caused by this 👍